### ISSUE TITLE
[Accessibility]A11y_VSCode_Feedback_Keyboard:"Close" button is not accessible through keyboard when NVDA is ON.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51900]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
Keyboard,NVDA[2018.2],JAWS[18.0.5038]
**Repro Steps:**
1. Launch VS Code.
2.Navigate to Status Bar and select "Tweet Feedback" button.
3. Check the keyboard navigation.

**Actual:**
"Close" button is not accessible through keyboard when NVDA is ON.

**Expected:**

"Close" button should accessible through keyboard.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
The keyboard user will not be able to access the control.
**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](url)

Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference:**
[[Accessibility]A11y_VSCode_Feedback_Keyboard2.pptx](https://github.com/Microsoft/vscode/files/2102221/Accessibility.A11y_VSCode_Feedback_Keyboard2.pptx)

**Comment by isidorn** [comment]
You can close the feedback widget using ESC.
@AccessibilityTestingTeam-TCS Why is that not enough?

**Comment by isidorn** [comment]
No. Pressing Esc once works perfectly fine both on Win and Mac for me.
